app/controllers/admin/groups_controller.rb

```diff
@@ -4,9 +4,20 @@ class Admin::GroupsController < ApplicationController
   before_action :find_event
 
   def index
+    @event_groups = @event.event_groups
   end
 
   def generate
+    @attendees = @event.applications.application_selected.confirmed
+    @attendees.each_slice(6).with_index do |group, index|
+      event_group = EventGroup.create(event: @event, name: "Group #{index}")
```

Comment:
Here we should use `Group #{index + 1}` since the `index` coming from the block variable starts at 0.